enqueue.go

```diff
@@ -62,12 +72,16 @@ func EnqueueWithOptions(queue, class string, args interface{}, opts EnqueueOptio
 
 	bytes, err := json.Marshal(data)
 	if err != nil {
-		return "", err
+		return "", 0, err
```

Comment:
An unsuspecting caller that doesn't check the err deserves to be fooled.

Seriously, the current code is idiomatic Go. `0` is the [zero value](https://golang.org/ref/spec#The_zero_value) for integer types. If `err` is non-nil, the caller is supposed to ignore the other return values.